walletkit_client.go

```diff
@@ -737,28 +742,92 @@ func (m *walletKitClient) ListSweepsVerbose(ctx context.Context,
 	return result, nil
 }
 
+// BumpFeeOption customizes a BumpFee call.
+type BumpFeeOption func(*walletrpc.BumpFeeRequest)
+
+// WithImmediate is an option for enabling the immediate mode of BumpFee. The
+// sweeper will sweep this input without waiting for the next batch.
```

Comment:
... without waiting for the next block.

Reply:
Fixed. Thanks!

Reply:
Does it make sense to fix the description of the field `Immediate` as well?
https://pkg.go.dev/github.com/lightningnetwork/lnd@v0.18.5-beta/lnrpc/walletrpc#BumpFeeRequest

Reply:
Opened a PR here to fix the description: https://github.com/lightningnetwork/lnd/pull/9518